### DISCUSSION TITLE
context-free-botの頻度を減らしたい

### DISCUSSION THREAD

**Discussion by hakatashi** [tsg-ut/slackbot#886] [Polls]
このトピックは GitHub Discussions を用いた投票です。

以下は @hakatashi による提案文です。

>sandbox発足以来、様々なslackbotが少しずつ追加され、現状のsandboxはBOTのメッセージの割合が高くなりすぎているように感じます。気軽にメッセージを書き込める場所として、sandboxにBOTなどの定期的な書き込みがあることは良いことですが、現状少し多すぎるように感じます。
とくに文脈を無視して意味のない情報を書き込むBOTであるよう設計されたcontext-free-botは会話を阻害する要素が多く、sandboxにとって無用の存在になっているように感じます。
そこで、context-free-botの投稿頻度を現状の10分の1程度の頻度とし、ごくまれに見かける程度にしたいです。

投票期限はこのメッセージがpostされてから1ヶ月以内とし、原則として賛成票と反対票の多い方を採用します。

補足意見がある場合はこのトピックにメッセージで返信してください。

**Comment by iLiss557** [comment]
Context Free Botに関しては議論の発展に貢献することも非常に稀である印象を受けます 廃止を含めて検討する余地があると思いますが、残すにしても1日に1回 (1/24?) くらいの頻度でもよい気がします

**Comment by hakatashi** [comment]
本件は 賛成11:反対0 で可決されました。

**Comment by hakatashi** [comment]
#893 にて変更反映済みです
